Lib/test/test_tarfile.py

```diff
@@ -3809,23 +3840,21 @@ def test_parent_symlink(self):
             arc.add('current', symlink_to='.')
 
             # effectively points to ./../
-            arc.add('parent', symlink_to='current/..')
+            if self.dotdot_resolves_early:
```

Comment:
This use of `dotdot_resolves_early` isn't guarded by `os_helper.can_symlink()`.
It might be better to always set `dotdot_resolves_early`.